components/store/StoreCard.js

```diff
@@ -55,89 +64,108 @@ export default function StoreCard({
     });
   };
 
+  const storeOpenStatus = computeStoreOpen(store.storeHours);
```

Comment:
I'm getting REALLY SLOW loading! Is there a way we can make this only call once? It seems to be called with every movement on the map and gets REALLY REALLY slow on the store list screen. When I type to search, it's so slow that it doesn't even respond at all. It gets to a point where I can't load all the stores on storelist and then it fully stops responding altogether like I can't tap map icons or close the Find a Store page.

Reply:
Yeah good point! I removed one call in the `StoreDetailsScreen` and pass the summary line in as a prop from `StoreCard`. Hopefully this helps!